### PR TITLE
adapt and get more information from nightly chain-sync bm

### PR DESCRIPTION
**Pull request by CodiePP**

### CODE REVIEW

File: .buildkite/full-sync-nightly.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ steps:
     command:
       - nix-shell -A devops --run "niv update cardano-node"
```
Comment (jbgi) [inline .buildkite/full-sync-nightly.yml:4]:
```suggestion
      - nix-shell -p niv --run "niv update cardano-node"
```
to be consistent.

File: .buildkite/nightly.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ steps:
     command:
       - nix-shell -A devops --run "niv update cardano-node"
```
Comment (jbgi) [inline .buildkite/nightly.yml:4]:
```suggestion
      - nix-shell -p niv --run "niv update cardano-node"
```
to be consistent.

**Review by jbgi — APPROVED** [review]
with two minor changes for consistency.

**Comment by CodiePP** [comment]
verified in run https://buildkite.com/input-output-hk/cardano-benchmarking-nightly/builds/91

**Pull request merged**